roles/build-release/tasks/build.yaml

```
@@ -39,10 +39,10 @@
     (antsibull_ansible_version is regex("^\d+.\d+.\d+(a\d+|b1)$") and (antsibull_build_reset | bool))
     or not _antsibull_build_file_stat.stat.exists
 
-- name: Set up feature freeze for b2 and later betas, and release candidates
+- name: Set up feature freeze for b2 and later betas, and release candidates (unless explicitly ignored by user)
   ansible.builtin.set_fact:
     _feature_freeze: "--feature-frozen"
-  when: antsibull_ansible_version is regex("^\d+.\d+.\d+(b[2-9]|rc\d+)$")
+  when: antsibull_ansible_version is regex("^\d+.\d+.\d+(b[2-9]|rc\d+)$") and not antsibull_ignore_feature_freeze
```

Nit: The line is a bit long as it is and I think this ordering makes more sense

```suggestion
  when: 
    - not antsibull_ignore_feature_freeze
    - antsibull_ansible_version is regex("^\d+.\d+.\d+(b[2-9]|rc\d+)$")
```